sample.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
The `sample.json` is ok, however it would be nice if we change the stylesheet URL into something else because the tool already uses Water.css by default